letsdrill.rb

```diff
@@ -1,24 +1,36 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  #Put your code here!
+  if integer.is_a? Integer
+    if integer>=90
+      "A"
+    elsif integer>=80
+      "B"
+    elsif integer>=70
+      "C"
+    elsif integer>=60
+      "D"
+    else
+      "F"
+    end
+  else
+    puts "Expected an Integer. You supplied #{integer}"
+  end
 end
 
 def shortest_string(array)
-
   #Put your code here!
-
+  array.sort! {|l,r| l.length <=> r.length}
+  array[0]
```

Comment:
Please use `min_by` instead. Sorting a array first in place causes too many data movements, that's not very desirable in usual circumstances.